### PR TITLE
F/cleanup

### LINKED ISSUE #48: An in-range update of history is breaking the build 🚨
## Version **4.5.1** of [history](https://github.com/mjackson/history) just got published.

<table>
  <tr>
    <th align=left>
      Branch
    </th>
    <td>
      <a href="/philschatz/gh-board/compare/greenkeeper%2Fhistory-4.5.1">Build failing 🚨</a>
    </td>
  </tr>
  <tr>
    <th align=left>
      Dependency
    </td>
    <td>
      history
    </td>
  </tr>
  <tr>
    <th align=left>
      Current Version
    </td>
    <td>
      4.5.0
    </td>
  </tr>
  <tr>
    <th align=left>
      Type
    </td>
    <td>
      dependency
    </td>
  </tr>
</table>

This version is **covered** by your **current version range** and after updating it in your project **the build failed**.

As history is a direct dependency of this project **this is very likely breaking your project right now**. If other packages depend on you it’s very likely also breaking them.
I recommend you give this issue a very high priority. I’m sure you can resolve this :muscle:

---

<details>
<summary>Status Details</summary>

- ❌ **continuous-integration/travis-ci/push** The Travis CI build failed [Details](https://travis-ci.org/philschatz/gh-board/builds/190450535)

</details>

<details>
<summary>Commits</summary>
<p>The new version differs by 4 commits .</p>
<ul>
<li><a href="https://urls.greenkeeper.io/mjackson/history/commit/abebe50801f60d11e4f73c0a9030368c2273d712"><code>abebe50</code></a> <code>Version 4.5.1</code></li>
<li><a href="https://urls.greenkeeper.io/mjackson/history/commit/817af4423e76fa71db95fd094881b39abbfd99fe"><code>817af44</code></a> <code>Fix unsubscribe bug</code></li>
<li><a href="https://urls.greenkeeper.io/mjackson/history/commit/5f634597fc1a252eac5edca23cae4823a6e591be"><code>5f63459</code></a> <code>Update babel</code></li>
<li><a href="https://urls.greenkeeper.io/mjackson/history/commit/2c7a038c8221ab50d6d2c8b27571eed414dc0119"><code>2c7a038</code></a> <code>Update CHANGES</code></li>
</ul>
<p>See the <a href="https://urls.greenkeeper.io/mjackson/history/compare/6e83bc3aaaaff5793ea2db48cf70f309f44acc53...abebe50801f60d11e4f73c0a9030368c2273d712">full diff</a>.</p>
</details>

<details>
<summary>Not sure how things should work exactly?</summary>

There is a collection of [frequently asked questions](https://greenkeeper.io/faq.html) and of course you may always [ask my humans](https://github.com/greenkeeperio/greenkeeper/issues/new).
</details>

---

Your [Greenkeeper](https://greenkeeper.io) Bot :palm_tree:

### PR DESCRIPTION
**Pull request by mathieudutour**
I'm going through the code, cleaning up code as I stumble on it.

- [x] fix #48
- [x] add a link to go back to the kanban on the nav bar even when there is only 1 repo (useful when going to the burnup view for example)
- [x] fix the IssueList if no primaryRepo is given (in Milestone view for example) (broken after https://github.com/philschatz/gh-board/commit/533c3a3e417d57e78875fe539505be1735e6f80f)

### CODE REVIEW

File: src/components/async-button.jsx [hunk 1]
```diff
@@ -46,7 +46,7 @@ export default React.createClass({
     }
   },
   render() {
-    const {children, waitingText, renderError, doneText, className} = this.props;
+    const {children, waitingText, renderError, doneText, className, ...rest} = this.props;
```
Comment (philschatz) [inline src/components/async-button.jsx]: oh nice! I didn't know about `babel-plugin-transform-object-rest-spread`

File: src/components/issue.jsx [hunk 2]
```diff
@@ -497,13 +497,11 @@ let Issue = React.createClass({
...
     // TODO: Not sure why React no longer automatically binds all functions to `this`
-    this._changeListener = this.forceUpdate.bind(this);
-    card.onChange(this._changeListener);
+    this._unsubscribe = card.onChange(this.forceUpdate.bind(this));
     Timer.onTick(this.pollPullRequestStatus);
   },
   componentWillUnmount() {
-    const {card} = this.props;
-    card.offChange(this._changeListener);
+    this._unsubscribe && this._unsubscribe();
```
Comment (philschatz) [inline src/components/issue.jsx]: Nice, more concise!

File: src/github-client.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ import jsonqueryEngine from 'jsonquery-engine';
 
 
 
-import Octo from 'octokat/dist/node/base';
+import Octo from 'octokat';
```
Comment (philschatz) [inline src/github-client.js]: I've been planning to reduce the size of Octokat by opting into plugins like the one that converts `snake_case` responses to `camelCase` (similar to `lodash`). I _think_ that was why I put this line in, but yep, nice catch!

Comment (mathieudutour) [inline src/github-client.js]: I wasn't sure, I was trying to fix the issue in the octokat. This change might not have been necessary.

**Pull request merged**